infra/containers.yaml

```diff
@@ -109,6 +109,14 @@ Resources:
               Value: !Sub '${AWS::Region}'
             - Name: AWS_S3_BUCKET_NAME
               Value: !Ref 'S3BucketName'
+            - Name: IDCS_CLIENT_ID
+              Value: '{{resolve:secretsmanager:dev-oidc-secrets:SecretString:IDCS_CLIENT_ID}}'
```

Comment:
Thoughts (non-blocking): the names `dev-xxx` suggest that these are not for prod use, maybe we should create ones that are only used in prod?

Reply:
That is definitely correct - but we only have dev secrets right now, this is to-be-updated when we get prod secrets for real.  That will be after our security review.